pkg/kubelet/prober/worker.go

```diff
@@ -251,7 +251,7 @@ func (w *worker) doProbe() (keepGoing bool) {
 	if c.Started != nil && *c.Started {
 		// Stop probing for startup once container has started.
 		if w.probeType == startup {
-			return false
+			return true
```

Comment:
Is the comment still true?

Reply:
Yeah.  Revert #98571 https://github.com/kubernetes/kubernetes/pull/98571/files#diff-37568d7080a94147e978aa46a84cf8c6a4493b6e79030f787df268cb7dc8ba2aR247

Reply:
Yes, it is still valid, since the return value tells the controller to keep running, but by returning early we skip the actual probe run.